tests/ui/conftest.py

```diff
@@ -32,35 +46,101 @@ def jwt_issuer(base_url, variables):
-    user = {
-        'email': fxa_account.email,
-        'password': fxa_account.password,
-        'username': fxa_account.email.split('@')[0]}
+def create_superuser(
+        transactional_db, live_server, base_url, tmpdir, variables):
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials=str(tmpdir.join('variables.json')),
+        hostname=urlparse.urlsplit(base_url).hostname
+    )
+
+    with tmpdir.join('variables.json').open() as f:
+        variables.update(json.load(f))
+
+
+@pytest.fixture
+def user(transactional_db, create_superuser, base_url,
+         fxa_account, jwt_token):
     url = '{base_url}/api/v3/accounts/super-create/'.format(base_url=base_url)
+
     params = {
-        'email': user['email'],
-        'username': user['username'],
-        'password': user['password'],
+        'email': fxa_account.email,
+        'password': fxa_account.password,
+        'username': fxa_account.email.split('@')[0],
         'fxa_id': fxa_account.session.uid}
     headers = {'Authorization': 'JWT {token}'.format(token=jwt_token)}
-    r = requests.post(url, data=params, headers=headers)
-    assert requests.codes.created == r.status_code
-    user.update(r.json())
+    response = requests.post(url, data=params, headers=headers)
+    user = params
+    assert requests.codes.created == response.status_code
+    user.update(response.json())
     return user
+
+
+@pytest.fixture(scope='function')
+def live_server(request):
+    """
+        This fixture overrides the live_server fixture provided by
+        pytest_django. live_server allows us to create a running version of the
+        addons django application within pytest for testing.
+
+        Christopher Grebs:
+        From what I found out was that the `live_server` fixture (in our setup,
+        couldn't reproduce in a fresh project) apparently starts up the
+        LiveServerThread way too early before pytest-django configures the
+        settings correctly.
+
+        That resulted in the LiveServerThread querying the 'default' database
+        which was different from what the other fixtures and tests were using
+        which resulted in the problem that the just created api keys could not
+        be found in the api methods in the live-server.
+
+        I worked around that by implementing the live_server fixture ourselfs
+        and make it function-scoped so that it now runs in a proper
+        database-transaction.
+
+        This is a HACK and I'll work on a more permanent solution but for now
+        it should be enough to continue working on porting tests...
+
+        Also investigating if there are any problems in pytest-django directly.
+    """
+
+    request.getfixturevalue('transactional_db')
```

Comment:
If `transactional_db` is required, you can just include it in the function arguments instead of using `getfixturevalue`.